<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaturalWidget</class>
 <widget class="QWidget" name="NaturalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>1143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>1123</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonNewNaturalCostSpace">
         <property name="text">
          <string>New Natural Cost Space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="HRICSNatural">
         <property name="title">
          <string>Natural Cost Space</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="checkBoxDrawNaturalColor">
               <property name="text">
                <string>Draw Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonDeleteNaturalCostSpace">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButtonComputeAllNatCost">
               <property name="text">
                <string>Recompute All Cost</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButtonComputeReachability">
               <property name="text">
                <string>Compute Reachability</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="checkBoxHriLeftArmVsRightArm">
               <property name="text">
                <string>Left Arm Vs Right Arm</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pushButtonGetSortedCells">
               <property name="text">
                <string>Get Sorted Cells</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButtonRecomputeReach">
               <property name="text">
                <string>Recompute All Reachability</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_18" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_NaturalParam">
               <property name="text">
                <string>Show Natural Parameters</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="NaturalParameters">
            <property name="title">
             <string>Natural Parameters</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QGridLayout" name="gridLayoutNatural">
               <item row="0" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Neutral</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxNeutralDist">
                 <property name="decimals">
                  <number>4</number>
                 </property>
                 <property name="maximum">
                  <double>20.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QSlider" name="horizontalSliderNeutralDist">
                 <property name="maximum">
                  <number>5000</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Energy</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxEnergy">
                 <property name="decimals">
                  <number>4</number>
                 </property>
                 <property name="maximum">
                  <double>20.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QSlider" name="horizontalSliderEnergy">
                 <property name="maximum">
                  <number>5000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Discomfort</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxDiscomfort">
                 <property name="decimals">
                  <number>4</number>
                 </property>
                 <property name="maximum">
                  <double>20.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QSlider" name="horizontalSliderDiscomfort">
                 <property name="maximum">
                  <number>5000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="toolTip">
                  <string>to the left (negatives) for left handed human, and right (positives) for right handed human</string>
                 </property>
                 <property name="text">
                  <string>Arm Pref</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxArmPref">
                 <property name="toolTip">
                  <string>to the left (negatives) for left handed human, and right (positives) for right handed human</string>
                 </property>
                 <property name="decimals">
                  <number>4</number>
                 </property>
                 <property name="minimum">
                  <double>-1.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.050000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QSlider" name="horizontalSliderArmPref">
                 <property name="toolTip">
                  <string>to the left (negatives) for left handed human, and right (positives) for right handed human</string>
                 </property>
                 <property name="maximum">
                  <number>5000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Loading Grids</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButtonLoadActiveGridFromFile">
               <property name="text">
                <string>Load Active Grid From File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSaveActiveGridToFile">
               <property name="text">
                <string>Save Active Grid To File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonMergeGrids">
               <property name="text">
                <string>Merge 1 And 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="activeGridsBox">
               <item>
                <property name="text">
                 <string>No Grid Selcted</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="pushButton_initColPos">
                  <property name="text">
                   <string>Init col and pos</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Show ith Cell</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCellToShow">
                  <property name="maximum">
                   <number>5000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_17" native="true">
            <layout class="QFormLayout" name="formLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::FieldsStayAtSizeHint</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButtonInitBaseGrid">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Initialization of base grid</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_gridProperties">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Show base grid properties</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QGroupBox" name="groupBox_gridPreperties">
               <property name="title">
                <string>Grid properties</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>x min :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_xmin">
                  <property name="text">
                   <string>-1.0</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>x max :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="label_xmax">
                  <property name="text">
                   <string>1.0</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QSlider" name="horizontalSlider_xmin">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>30</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>true</bool>
                  </property>
                  <property name="invertedControls">
                   <bool>false</bool>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::NoTicks</enum>
                  </property>
                  <property name="tickInterval">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2" colspan="2">
                 <widget class="QSlider" name="horizontalSlider_xmax">
                  <property name="maximum">
                   <number>30</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>y min :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="label_ymin">
                  <property name="text">
                   <string>-1.0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>y max :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QLabel" name="label_ymax">
                  <property name="text">
                   <string>1.0</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QSlider" name="horizontalSlider_ymin">
                  <property name="maximum">
                   <number>30</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2" colspan="2">
                 <widget class="QSlider" name="horizontalSlider_ymax">
                  <property name="maximum">
                   <number>30</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>z min :</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLabel" name="label_zmin">
                  <property name="text">
                   <string>0.5</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>z max :</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="3">
                 <widget class="QLabel" name="label_zmax">
                  <property name="text">
                   <string>2.5</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QSlider" name="horizontalSlider_zmin">
                  <property name="minimum">
                   <number>-10</number>
                  </property>
                  <property name="maximum">
                   <number>30</number>
                  </property>
                  <property name="value">
                   <number>-5</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2" colspan="2">
                 <widget class="QSlider" name="horizontalSlider_zmax">
                  <property name="minimum">
                   <number>10</number>
                  </property>
                  <property name="maximum">
                   <number>40</number>
                  </property>
                  <property name="value">
                   <number>25</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
